django/core/management/commands/sendtestemail.py

```diff
@@ -0,0 +1,18 @@
+import socket, datetime
+
+from django.core.management.base import BaseCommand, CommandError
+from django.core.mail import send_mail
+
+class Command(BaseCommand):
+    help = "Sends an email through the framework to an address specified on the command line."
+    args = "<email email...>"
```

Comment:
Perhaps it would be clearer to say `<email1 email2...>` to clarify that multiple addresses are allowed?